docker.py

```diff
@@ -963,6 +979,14 @@ def get_scipy_choices():
     include_dir='/usr/include',
     lib_dir='/usr/lib/x86_64-linux-gnu',
 )
+codes['nccl2.9-cuda114'] = nccl_base.format(
+    nccl_sub_dir='cuda',
+    nccl_os_ver='1804',
+    libnccl2='libnccl2_2.9.9-1+cuda11.3_amd64',
+    libnccl_dev='libnccl-dev_2.9.9-1+cuda11.3_amd64',
```

Comment:
Seems that NCCL provides a binary for each minor version, and the one for CUDA 11.4 is not put yet. Skip `'nccl2.9-cuda114'` this time?

Reply:
I think it's ok to do this for now.

Reply:
Raised an issue for record #675